docs/lib/DEBUGGING.md

```diff
@@ -144,6 +144,14 @@ The OpenSDA port provides a debugger interface, which is compatible with the Seg
 
 To use the JTAG port instead, download the tools and drivers for the debugger of your choice.
 
+#### ULINK pro configuration for NXP FRDM-K64F
+
+Configure NXP FRDM-K64F with the pdf document from arm Keil [Application Note](http://www.keil.com/appnotes/docs/apnt_287.asp).
+
+In order to add an example to uvision run the "mbed export -i UVISION5 -m K64F" command, and open the project from uvision with the .uvprojx file created.
+
+Most ULINK pro features configuration are described in the Application Note except Instruction trace that is not officially supported with cortex-M devices, we can bypass that with the PC SAMPLING option (options for target menu, Debug tab, go to Setting, Trace tab), choose a Prescaler so that Period will be shorter then core clock time (1/frequency), Mark the “Periodic” and approve.
```

Comment:
1. All the description in the braces Can be written as bullets).
1. It will be much more informative to add an image that shows those steps.